config/custom-environment-variables.yaml

```diff
@@ -38,6 +38,19 @@ httpd:
 
 datastore:
     plugin: DATASTORE_PLUGIN
+    sequelize:
+        # Type of server to talk to
+        dialect: DATASTORE_SEQUELIZE_DIALECT
+        # Database name
+        database: DATASTORE_SEQUELIZE_DATABASE
```

Comment:
Nvm it's already default here: https://github.com/screwdriver-cd/datastore-sequelize/blob/master/index.js#L126